backend/src/main/resources/freinet_import/categories.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Don't we have an endpoint that we could fetch from instead of serving this statically? (Although it doesn't really matter)

Reply:
No idea. @maxammann is there a way to see all files that can be queried from the freinet api?

Reply:
Not that I know of. This seems fine!

Reply:
But thee ids seems wrong. They go from 0 to 8 in NRW.

Reply:
Freinet api returns values from 1-9 (inclusive), I got the values from here: https://github.com/ehrenamtskarte/artefacts/blob/main/api_access.md